gdal-sys/build.rs

```diff
@@ -114,41 +119,109 @@ fn main() {
     let mut version = env::var_os("GDAL_VERSION")
         .map(|vs| vs.to_string_lossy().to_string())
         .and_then(|vs| Version::parse(vs.trim()).ok());
+    let use_vcpkg = if cfg!(windows) {
+        env::var_os("GDAL_VCPKG").is_some()
+    } else {
+        false
+    };
 
     let mut found = false;
     if cfg!(windows) {
-        // first, look for a static library in $GDAL_LIB_DIR or $GDAL_HOME/lib
-        // works in windows-msvc and windows-gnu
-        if let Some(ref lib_dir) = lib_dir {
-            let lib_path = lib_dir.join("gdal_i.lib");
-            if lib_path.exists() {
-                prefer_static = true;
-                lib_name = String::from("gdal_i");
-                found = true;
+        if use_vcpkg {
+            let vcpkg_root = env_dir("VCPKG_ROOT");
+            let vcpkg_triplet = env::var("GDAL_VCPKG_TRIPLET");
+
+            if vcpkg_root.is_none() {
+                panic!("GDAL_VCPKG requires VCPKG_ROOT to be set.");
             }
-        }
-        if !found {
-            if let Some(ref home_dir) = home_dir {
-                let home_lib_dir = home_dir.join("lib");
-                let lib_path = home_lib_dir.join("gdal_i.lib");
+
+            if vcpkg_triplet.is_err() {
+                panic!("GDAL_VCPKG requires GDAL_VCPKG_TRIPLET to be set.");
+            }
+
+            let vcpkg_root = vcpkg_root.unwrap();
```

Comment:
```suggestion
            let vcpkg_root = vcpkg_root.expect("GDAL_VCPKG requires VCPKG_ROOT to be set");
```

(same for `vcpkg_triplet`)